plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/ConnectionProperties.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.core.connection;
+
+/**
+ * Property IDs for handling connection change events
+ * 
+ * @author jeff.cantrill
+ *
+ */
+public interface ConnectionProperties {
```

Comment:
I would call this accoring to the general naming pattern for interfaces in Eclipse IConnectionProperties